goodkey/good_key.go

```diff
@@ -186,6 +195,11 @@ func (policy *KeyPolicy) goodKeyRSA(key rsa.PublicKey) (err error) {
 	if !policy.AllowRSA {
 		return berrors.MalformedError("RSA keys are not allowed")
 	}
+	if policy.weakRSAList != nil {
+		if policy.weakRSAList.Known(key.N.Bytes()) {
```

Comment:
Could you collapse these two `if` blocks into one with the condition `policy.weakRSAList != nil && policy.weakRSAList.Known(key.N.Bytes())` ?